src/components/ProjectList/listOfProjects.js

```diff
@@ -635,5 +635,12 @@ const projectList = [
     description: 'Column-oriented database management system that allows generating analytical data reports in real time.',
     tags: ['C++', 'C++20', 'cpp', 'Linux', 'DBMS', 'OLAP', 'Analytics', 'SQL', 'Big Data', 'Distributed Database', 'MPP'],
   },
+  {
+    name: 'RDoc::Generator::SolarFish',
```

Comment:
Hey, can we have the display name as `RDoC-Generator-SolarFish` instead of the `::`

Reply:
No problem. But I just checked and it appeared that `RDoc::Generator::SolarFish` or `RDoc-Generator-SolarFish` is too long for current layout and looks like this:

![image](https://user-images.githubusercontent.com/8436629/73183490-ecc5b300-412b-11ea-93e3-f8cc972d2c8d.png)

So I renamed it to `RDoc SolarFish`.